packages/react-query/src/QueryAsyncBoundary.tsx

```diff
@@ -44,6 +44,19 @@ if (process.env.NODE_ENV !== 'production') {
 /**
  * This component wrapping QueryErrorResetBoundary of @tanstack/react-query with @suspensive/react's AsyncBoundary.
  *
+ * So you must install @suspensive/react first, then use it.
+ * ```shell
+ * npm install @suspensive/react @suspensive/react-query
+ * ```
+ *
+ * ```shell
+ * pnpm add @suspensive/react @suspensive/react-query
+ * ```
+ *
+ * ```shell
+ * yarn add @suspensive/react @suspensive/react-query
+ * ```
+ *
```

Comment:
Users watching this `ts-doc` already have `@suspensive/react-query` installed, so wouldn't it be enough to show only the `@suspensive/react` installation phrase?

```suggestion
 * So you must install @suspensive/react first, then use it.
 * ```shell
 * npm install @suspensive/react
 * ```
 *
 * ```shell
 * pnpm add @suspensive/react
 * ```
 *
 * ```shell
 * yarn add @suspensive/react
 * ```
 *
```

Reply:
👍👍👍